src/Barcode/EanUpc.php

```diff
@@ -35,6 +35,10 @@ public function __construct($code, $length, $leftMargin, $rightMargin, $xDim, $b
 	 */
 	private function init($code, $length)
 	{
+		if (preg_match('/[\D]+/', $code)) {
+			throw new \Mpdf\Barcode\BarcodeException('Invalid EAN UPC barcode value');
```

Comment:
Throwing an exception would mean this should be postponed for 9.0. What happens when the exception is not thrown?